### PR TITLE
fix: added video-filter

### LINKED ISSUE #18: Issue: added category buttons
added category filter buttons in videopage. User can filter categories of videos according to the need.

### PR DESCRIPTION
**Pull request by RahulSingh9131**
Added filter videos according to category .

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
- all functionality work perfectly working
- good use of context avoid prop drilling 
-code looks clean and good
good to merge

File: src/css/screens/videopage.css [hunk 1]
```diff
@@ -24,4 +24,20 @@
 }
 .videopage-body{
     margin-top: var(--mb-2-5);
+}
+/* .tabs-container{
```
Comment (vaibhav-mahalle) [inline src/css/screens/videopage.css]: Remove commented code while raising PR.

File: src/pages/VideoPage.js [hunk 2]
```diff
@@ -8,34 +7,35 @@ import Footer from '../components/Footer';
...
+            <button className='category-tab' onClick={()=>videoDispatch({type:"CATEGORY_FILTER",payload:videos})}>All</button>
+
+        </div>
         <section className='videopage-body'>
           <div className='container'>
             <Sidebar/>
             <div className='flex flex-wrap'>
-              {video && video.map(({_id,image,title,timestamp,views,category})=>{
+              {(filteredList.length!=0 ?  filteredList:videos).map(({_id,image,title,timestamp,views,category})=>{
                 return (
```
Comment (vaibhav-mahalle) [inline src/pages/VideoPage.js:39]: The formatting of the code is very good.

**Review by vaibhav-mahalle — APPROVED** [review]
Added some suggestions to the PR, continue the good working.

**Pull request merged**